testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
@stanley-cheung Yes, we should add this to composer. Also, are you sure the override is required in the userland `composer.json`? If so, that seems like a bug in composer. And also if so, we could submit your fork as a new composer package, which would definitely fix the issue.

Requiring the users to override in `composer.json` rather than just call `composer require grpc/grpc` is not acceptable DX IMHO. We could fork to `GoogleCloudPlatform` and submit a new package from there.